src/main/java/ch/njol/skript/classes/data/BukkitEventValues.java

```diff
@@ -1935,5 +1933,36 @@ public RegainReason get(EntityRegainHealthEvent event) {
 				return event.getRegainReason();
 			}
 		}, EventValues.TIME_NOW);
+
+		// BeaconEffectEvent
+		if (Skript.classExists("com.destroystokyo.paper.event.block.BeaconEffectEvent")) {
+			EventValues.registerEventValue(BeaconEffectEvent.class, PotionEffectType.class, new Getter<PotionEffectType, BeaconEffectEvent>() {
+				@Override
+				public @Nullable PotionEffectType get(BeaconEffectEvent event) {
```

Comment:
```suggestion
				public PotionEffectType get(BeaconEffectEvent event) {
```
geteffect and gettype are explicitly NotNull
if you can guarantee the get method never returns null, remove the nullable annotation
this applies to the rest of these event values too